src/utils.js

```diff
@@ -188,9 +188,18 @@ var validateGroupName = function validateGroupName(key, groupName) {
         '. Please use strings or array of strings for groupName');
 };
 
+// parses the value of a url param (for example ?gclid=1234&...)
+var getQueryParam = function getQueryParam(name, query) {
+  name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+  var regex = new RegExp("[\\?&]" + name + "=([^&#]*)");
```

Comment:
Seems ok but I always get a bit nervous with regex parsing.  Is it worth taking into account the possibility of repeated params?